### ISSUE TITLE
BasePicker component should support customizing RemoveButton aria label for each item.

### ISSUE THREAD

**Issue by zeedomD** [microsoft/fluentui#14776]
### Environment Information

- **Package version(s)**: 7.131.0
- **Browser and OS versions**: Not Related

### Describe the issue:

BasePicker already provide an API “RemoveButtonAriaLabel” for Screen reader when focusing on the "Remove Button" on a Picked Item.
But we think it would be better to make it customizable based on each item. So that we can make the aria label like "Remove XXXXXX".

![image](https://user-images.githubusercontent.com/20253660/91414629-69b23a00-e87f-11ea-9978-1e90f88c8799.png)

### Please provide a reproduction of the issue in a codepen:

No Reproduction steps are required.

#### Actual behavior:

We can only specific a general aria label for all picker items.

#### Expected behavior:

We should specific customized aria label for each picker items.

### Documentation describing expected behavior

N/A

**Comment by paulgildea** [comment]
@zeedomD Thanks for the feedback!

Sounds like this more a feature request rather than an unexpected behavioral issue with the current Picker API.  mark this as a feature request and add it to our backlog, but I don't have any timelines to share at this time as we've been focused on converging our two react libraries. We'll take this at input when we start to converge our `Picker` solutions.

@joschect - Do you have any work around ideas for this? Is there a custom rendering interface that might provide this functionality?

Thanks!

**Comment by joshmccall221** [comment]
Workaround / Fix: 
From [PeoplePickerAccessibility fixes ](#2286)(https://github.com/microsoft/fluentui/commit/5c9d10ee2cb1e4ccc82d5f03dccb02bf1276a23e)

```
    <CompactPeoplePicker
        removeButtonAriaLabel={'Remove'} 
```

**Comment by smhigley** [comment]
Fixed in #17723 :)

The remove button is now labelled by both the `removeButtonAriaLabel` and the text of the selected item. You can also now pass in a `selectionAriaLabel`, e.g. `selectionAriaLabel="Selected contacts"` to label the list of selected items as a whole.
